curriculum/challenges/english/25-front-end-development/quiz-css-positioning/66ed8ffcf45ce3ece4053eb5.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 ---
 
-Placeholder distractor 2
+When its parent element has `z-index: auto` and `position: fixed`.
 
 ---
 
-Placeholder distractor 3
+When it is combined with `float: left`, if two sibling elements are of the same `z-index` and absolutely positioned.
 
 #### --answer--
 
-Placeholder answer
+In combination with `position: fixed` or `position: sticky`.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+If two sibling elements have the same `z-index` and are both positioned absolutely using `position: absolute`, what determins which one is on top?
 
 #### --distractors--
 
-Placeholder distractor 1
+It rearranges the sibling elements in relation to its new position.
 
 ---
 
-Placeholder distractor 2
+Adding `z-index: auto` to the parent container.
 
 ---
 
-Placeholder distractor 3
+Using `position: relative` on the parent container.
 
 #### --answer--
 
-Placeholder answer
+The document source order (HTML position).
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+In what way will the property `position: relative` exert an effect on sibling elements in regards to document flow?
 
 #### --distractors--
 
-Placeholder distractor 1
+It removes the element from the normal document flow, which affects the positions of siblings.
 
 ---
 
-Placeholder distractor 2
+The relative element will move by the parent's top and left properties. 
 
 ---
 
-Placeholder distractor 3
+It creates a new stacking context which lifts the element in relation to any siblings.
 
 #### --answer--
 
-Placeholder answer
+It allows the element to shift without affecting the document flow of sibling elements.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What happens when a `position: relative` element is nested inside of a `position: absolute` parent, with top and left values set?
 
 #### --distractors--
 
-Placeholder distractor 1
+It moves relative to the parent element's original position.
 
 ---
 
-Placeholder distractor 2
+It moves with respect to the absolute coordinates of the containing parent element.
 
 ---
 
-Placeholder distractor 3
+It will never create a new stacking context whatever the `z-index` value.
 
 #### --answer--
 
-Placeholder answer
+It ignores the positioning of the parent and moves relative to its normal position.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the default of an element with `position: sticky` in terms of stacking context when combined with a `z-index` value?
 
 #### --distractors--
 
-Placeholder distractor 1
+It will create a new stacking context, and on top of any other element whose `z-index` is auto.
 
 ---
 
-Placeholder distractor 2
+By default it behaves like `position: absolute` creating a new stacking context.
 
 ---
 
-Placeholder distractor 3
+It ignores stacking contexts completely and will always end behind every element.
 
 #### --answer--
 
-Placeholder answer
+It stays within the stacking context in which it is if `z-index` is not explicitly defined.
```

Comment:
> if `z-index` is not explicitly defined.

Doesn't this contradict the question?

> when combined with a `z-index` value?

Reply:
Again, this is the question:

> What is the default of an element with `position: sticky` in terms of stacking context **when combined with a `z-index` value?**

And this is the answer:

> It stays within the stacking context in which it is **if `z-index` is not explicitly defined.**

How can it be both "when combined with a `z-index` value?" and "`z-index` is not explicitly defined."